digest/digest.go

```diff
@@ -2,7 +2,6 @@ package digest
 
 import (
 	"bytes"
-	"crypto/sha256"
```

Comment:
The changes in this file are meant to get rid of this import. I'd like to have knowledge about what the canonical digest algorithm is all in one place: `digester.go`